### ISSUE TITLE
repeat tracker initialize?

### ISSUE THREAD

**Issue by laisimiao** [MCG-NJU/MixFormer#20]
First of all, I thanks for your clean and high-quality codes. 
But in 
https://github.com/MCG-NJU/MixFormer/blob/219bd14704ec217919c3b1eb310940769546c2d6/external/AR/pytracking/VOT2020_super_only_mask_384_HP/mixformer_alpha_seg_class.py#L32-L43
I find two times of **tracker.initialize**. I think `initialize` is just a setting step (not online update step), why do we need two times of that?

**Comment by yutaocui** [comment]
That's the code for evaluation on VOT2020, which use Alpha-Refine [https://github.com/MasterBin-IIAU/AlphaRefine](url) as the second model to generate mask. As a result, we simultaneously initialize mixformer and alpha-refine.

**Comment by laisimiao** [comment]
yes. But `self.tracker.initialize` stands for initialization of mixformer and `self.alpha.initialize` stands for initialization of AR. So the codes do the repeat initialization of mixformer?

**Comment by yutaocui** [comment]
Sorry for that, we forgot commenting the line34-35 (\xF0\x9F\x98\x85).

**Comment by yutaocui** [comment]
Thanks for your reminder. We've updated it just now.
